src/lib/drivers/driver.js

```diff
@@ -232,10 +238,10 @@ class DriverBase {
 
   _readTraceFromStream(streamHandle) {
     return new Promise((resolve, reject) => {
-      // With our stream we can read a bunch, and if its taking too long,
-      // take a break to the next event cycle and then go again.
-      let isEOF = false;
-      let result = '';
+      // COMPAT: Chrome 50's implementation of `let` had some bugs which
+      // get triggered in this scenario. So we use let here.
+      var isEOF = false;
```

Comment:
OK
 i'll use `let` but leave the comment in case we miraculously have this as our only m50 blocker.